server/data/model/caseNotesHistory.ts

```diff
@@ -18,5 +18,5 @@ export type CaseNote = {
 
 export type CaseNotesHistory = {
   error?: string
-  results?: CaseNote[]
+  results?: CaseNote
```

Comment:
This is currently working even tough it's technically wrong as the compiled javascript isn't strongly typed and neither is the nunjucks which is does not use this as an array...